spring-cloud-contract-verifier/src/test/java/org/springframework/cloud/contract/verifier/builder/XmlBodyVerificationBuilderTest.java

```diff
@@ -0,0 +1,37 @@
+package org.springframework.cloud.contract.verifier.builder;
+
+import org.junit.Test;
+import org.springframework.cloud.contract.spec.Contract;
+import org.springframework.cloud.contract.spec.internal.BodyMatchers;
+
+import java.util.Optional;
+
+import static com.toomuchcoding.jsonassert.JsonAssertion.assertThat;
+
+public class XmlBodyVerificationBuilderTest {
+
+	private static final String xml = "<customer>\r\n"
+			+ "      <email>customer@test.com</email>\r\n" + "    </customer>";
+
+	@Test
+	public void shouldAddXmlProcessingLines() {
+		// Given
+		XmlBodyVerificationBuilder builder = new XmlBodyVerificationBuilder(
+				new Contract(), Optional.of(";"));
+		BlockBuilder blockBuilder = new BlockBuilder(" ");
+		BodyMatchers matchers = new BodyMatchers();
+		// When
+		builder.addXmlResponseBodyCheck(blockBuilder, xml, matchers, xml, true);
+		// Then
+		String test = blockBuilder.toString();
+		assertThat(test).contains(
```

Comment:
Just FYI you can chain those contains into one.

Reply:
done